relentless/simulate/lammps.py

```diff
@@ -512,107 +506,87 @@ def to_commands(self, sim):
 
         return cmds
 
-class AddNPTIntegrator(LAMMPSOperation):
-    r"""NPT integration via Nos\'e-Hoover thermostat/barostat.
+class Thermostat:
+    def __init__(self, T):
+        self.T = T
 
-    Parameters
-    ----------
-    dt : float
-        Time step size for each simulation iteration.
-    tau_T : float
-        Coupling constant for the thermostat.
-    tau_P : float
-        Coupling constant for the barostat.
+class BerendsenThermostat(Thermostat):
+    def __init__(self, T, tau):
+        super().__init__(T)
+        self.tau = tau
 
-    """
-    def __init__(self, dt, tau_T, tau_P):
-        self.tau_T = tau_T
-        self.tau_P = tau_P
+class NoseHooverThermostat(Thermostat):
+    def __init__(self, T, tau):
+        super().__init__(T)
+        self.tau = tau
 
-        self._fix = super().new_fix_id()
+class MTKThermostat(Thermostat):
+    def __init__(self, T, tau):
+        super().__init__(T)
+        self.tau = tau
```

Comment:
There is no `MTKThermostat` afaik (it is usually just a variation of Nose-Hoover), so you can remove this.

Reply:
I tried to include the MTK barostat-thermostat mentioned here `https://hoomd-blue.readthedocs.io/en/stable/module-md-integrate.html#hoomd.md.integrate.npt`

Let me know what to do for this case